html/indexbase.html

```diff
@@ -220,10 +226,43 @@ <h3 class="panel-title">Legend</h3>
             trigger: "hover",
             placement: "auto"
         })
-
     }
 }
 
+function updateRacks(statusObject) {
+    var nrHostsPerRow = 2;
+    racksNumbers = _.reduce(statusObject.hosts, function(racksNumbers, host) {
```

Comment:
this is a stranger reduce method - you don't need to push to rackNumbers explicitly.